lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.js

```diff
@@ -281,12 +281,16 @@ export default function BasicLTILaunchApp() {
       clientRPC.off('annotationActivity', onAnnotationActivity);
 
     /**
-     * @param {string} eventType
-     * @param {object} data
-     *   @param {string} data.date
+     * @param {import('../services/client-rpc').AnnotationEventType} eventType
+     * @param {import('../services/client-rpc').AnnotationEventData} data
      */
     function onAnnotationActivity(eventType, data) {
-      reportSubmission(data.date).then(unsubscribe);
+      if (
+        ['create', 'update'].includes(eventType) &&
```

Comment:
This guard wasn't here before (event type) because the app "should" only be configured to receive notifications on `create` and `update` events. However, it's both safer and more clear to future devs to be explicit here.